### PR TITLE
Update transport to use network module - Closes #2877

### LINKED ISSUE #2877: Update transport module in chain to use network module and fix the tests
### Description
Update the module/transport.js in chain module to use new network module created.
Also, it should remove the unused functions, and fix/remove tests related to the transport.

### Dependent issue
#2874

### PR DESCRIPTION
**Pull request by shuse2**
### Description
Update transport module to use the newly created network module

### Review checklist

* The PR resolves #2877
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/src/modules/chain/api/ws/transport.js [hunk 1]
```diff
@@ -28,14 +28,10 @@ const slaveRPCStub = require('./rpc/ws_rpc').slaveRPCStub;
  */
 function TransportWSApi(transportModule) {
 	wsRPC.getServer().registerRPCEndpoints({
-		updatePeer: transportModule.internal.updatePeer,
```
Comment (diego-G) [inline framework/src/modules/chain/api/ws/transport.js]: Where is the replacement of this?

File: framework/src/modules/chain/chain.js [hunk 2]
```diff
@@ -224,28 +224,9 @@ module.exports = class Chain {
...
-								transaction: transaction.id,
-								signatures: transaction.signatures,
-							};
-						}
-						return null;
-					})
-					.filter(transaction => transaction !== null);
-				return multisignatures;
-			},
+				promisify(this.scope.modulles.transport.shared.getTransactions)(null),
```
Comment (diego-G) [inline framework/src/modules/chain/chain.js]:
```suggestion
				promisify(this.scope.modules.transport.shared.getTransactions)(null),
```

File: framework/src/modules/chain/chain.js [hunk 3]
```diff
@@ -224,28 +224,9 @@ module.exports = class Chain {
...
+			getSignatures: async () =>
+				promisify(this.scope.modulles.transport.shared.getSignatures)(null),
```
Comment (diego-G) [inline framework/src/modules/chain/chain.js]:
```suggestion
				promisify(this.scope.modules.transport.shared.getSignatures)(null),
```

**Comment by diego-G** [comment]
I keep having issues when launching the node. Btw, the version of the P2P library is obsoleted. There is a commit in another PR to solve this https://github.com/LiskHQ/lisk/pull/3290/commits/c09cdd90b21345156af18d8d4e6147d4f62ae945
I recommend to rebase this branch after we merge #2881.

File: framework/test/mocha/common/generatePeerHeader.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+// Initialize global variables
+require('../setup');
+
+const randomstring = require('randomstring');
+
+const testConfig = __testContext.config;
+const random = require('./utils/random');
+
+const generatePeerHeader = function(headers = {}) {
```
Comment (jondubois) [inline framework/test/mocha/common/generatePeerHeader.js]: This is low priority but the name `generateNetworkConfig` or `generateP2PConfig` would be closer to new convention but then the file should be renamed too.

**Pull request merged**